test/e2e/azure_lb.go

```diff
@@ -129,11 +130,6 @@ func AzureLBSpec(ctx context.Context, inputGetter func() AzureLBSpecInput) {
 					Port:     80,
 					Protocol: corev1.ProtocolTCP,
 				},
-				{
-					Name:     "https",
```

Comment:
why this change?

Reply:
I think that's a holdover from when I was trying to fix the LB e2e spec failure, and since we don't use the HTTPS port here I thought to simplify things.

Reply:
let's either remove it or make it a separate commit (in this PR) to make it clear there was a behavior change beyond logging

Reply:
I removed that change (restored the https:// port).